packages/babel-helper-transform-fixture-test-runner/src/index.js

```diff
@@ -118,6 +118,7 @@ function runExec(opts, execCode) {
     transform: babel.transform,
     opts,
     exports: {},
+    require
```

Comment:
The problem with that is that the `require` will be relative to the `babel-helper-transform-fixture-test-runner/lib` folder instead of the test fixture.

`runExec` will need to receive a third argument with a "filepath" to use, and the value of `require` could be something like:

```js
require: id => require(id.startsWith('.') ? path.resolve(path.dirname(filepath), id) : id)
```

If not implementing this, then at least `throw` when the require starts with `.`, since otherwise can't possibly behave correctly.